packages/elements/src/Text/utils/common/index.ts

```diff
@@ -46,6 +46,15 @@ export function shouldExtractElement(
 ): boolean {
   const isAllowed = allowedTags.includes(element.tagName);
 
+  const isUlElement = element.tagName === "UL";
+  const hasDropdownMenuClass = element.classList?.contains("dropdown-menu");
+  const hasCountryListClass = element.classList?.contains("country-list");
+
+  if (isUlElement && !hasDropdownMenuClass && !hasCountryListClass) {
+    const htmlElement = element as HTMLElement;
+    htmlElement.style.marginTop = "15px";
+  }
```

Comment:
rollback issue @zeleniucvladislav would be fix here https://github.com/bagrinsergiu/MB-Support/issues/284#issue-2131438829